file-mon/src/main/java/com/wipro/ats/bdre/filemon/FileMonitor.java

```diff
@@ -98,10 +100,9 @@ private static void putEligibleFileInfoInMap(String fileName, FileContent fc) {
         // *Start*   Eligible files moved to data structure for ingestion to HDFS
         FileCopyInfo fileCopyInfo = new FileCopyInfo();
         try {
-          //  String exactFileName = fc.getFile().getName().getBaseName();
             fileCopyInfo.setFileName(fileName);
             fileCopyInfo.setSubProcessId(FileMonRunnableMain.getSubProcessId());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "Integer.toString" instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS2131)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Call the static method Integer.toString(...) instead of instantiating a temporary object to perform this to string conversion. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1158)